### ISSUE TITLE
Board position layout is reset on restart

### ISSUE THREAD

**Issue by jamestansx** [joplin/plugin-kanban#2]
*Set Board Layout Feature*

Currently, the board will always open horizontally at the right side at the initial. The position could be set by changing it at `View` > `Change application layout`. However, if changing to other notebooks with empty notes/todos in them, the kanban board will close. 

If `Joplin` was closed (shutdown), the board layout will be reset to the default unless the board is still opened when `Joplin` is closed. [note tab](https://github.com/benji300/joplin-note-tabs#:~:text=Tabs%20above%20note%20content) has this functionality that it can set the tab to be horizontal or vertical. 

TL;DR: feature requested to allow layout to be set to `horizontal` or `vertical` with closing Joplin does not affect the position of layout. 

Thanks a lot,
James

**Comment by mablin7** [comment]
Hm, I thought this should already work, but haven't tested it myself. So this is rather a bug then a feature request. Anyway, thanks for the report, I'll look into it soon

**Comment by jamestansx** [comment]
Ok, here's the detail of the bug:

*Steps to reproduce:*
1. Open up a kanban board
2. Position it to horizontal layout
3. Goto another note (closed the kanban board)
4. shutdown Joplin


*Expected behaviour:*

When starting Joplin again and open up the kanban board, it should be able to open in the same layout as previous (horizontal).


*Actual behaviour:* 

The position is reset to vertical layout.

**Comment by mablin7** [comment]
Fixed by 5836ec479b536bdc902bd1a14c0e48c89f28b077
